docs/unified-test-documentation/dasharo-performance/405-fan-control.md

```diff
@@ -0,0 +1,165 @@
+# Dasharo Performance: Fan Control
+
+## Test cases
+
+### Common
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+### FAN001.001 Fan does not stuck after coldboot (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that fan is not stuck at initial frequency after
+coldboot
+
+**Test configuration data**
+
+1. `FIRMWARE` = coreboot
+1. `OPERATING_SYSTEM` = `Ubuntu 22.04`
+
+**Test setup**
+
+1. Proceed with the [Common](#common) section.
+
+**Test steps**
+
+1. Disconnect DUT from power surce.
+1. Reconnect DUT to power source.
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Check if package `lm-sensors` is installed and if not, use bellow command in
```

Comment:
Maybe it will be a better option to add `lm-sensors` package checking to the test setup section